test/cpp/cpp_test_util.cpp

```diff
@@ -37,10 +38,15 @@ void ForEachDevice(const std::function<void(const Device&)>& devfn) {
   devfn(Device(default_device));
 }
 
-void AllClose(at::Tensor tensor, at::Tensor xla_tensor, double rtol,
-              double atol) {
-  EXPECT_TRUE(
-      ToCpuTensor(xla_tensor).allclose(ToCpuTensor(tensor), rtol, atol));
+bool TensorsEquals(at::Tensor tensor1, at::Tensor tensor2, double rtol,
```

Comment:
`TensorAreEqual`

Reply:
I used CloseValues() and we have EqualValue just above.